doc/extending/extending_theano.txt

```diff
@@ -876,11 +876,24 @@ Modify and execute the example to return two outputs: x + y
     and x - y.
 
 
-Documentation
--------------
+.. _Documentation:
 
-See :ref:`metadocumentation`, for some information on how to generate
-the documentation.
+Documentation and Coding Style
+------------------------------
+
+* The code should be compatible with Python 2.6 and above, as well as Python 3.3 and above (using 2to3 for conversion
+  to Python 3.x).
```

Comment:
Actually, we are not using `2to3` any more, we need the code to be natively compatible with Python 2 and Python 3, using `six` if needed.